src/components/Books/Book.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const { name, author } = props;
+
+  return (
+    <div className="book">
+      {name}
+      {author}
+    </div>
+  );
```

Comment:
Please add a button for removing books to this component according to the project requirement. The button wouldn't remove the books yet, this is just a placeholder for now.